scripts/pi-hole/js/queries.js

```diff
@@ -8,6 +8,22 @@
 /* global moment:false, utils:false */
 
 var tableApi;
+var tableFilters = [];
```

Comment:
Do the newly added variables really need to be global?

Reply:
Yes, it needs to be available all the time.